ops/charm.py

```diff
@@ -548,6 +548,10 @@ class RelationBrokenEvent(RelationEvent):
     are currently known locally.
     """
 
+    def defer(self) -> None:
+        """Relation-broken events are not deferrable."""
+        raise RuntimeError('Cannot defer relation-broken events')
```

Comment:
Would it be feasible to use Protocol for this so that calling `defer` becomes a static error?